schema.sql

```diff
@@ -6,7 +6,23 @@
     escape_attempts INTEGER,
     neutered BOOLEAN,
     weight_kg DECIMAL(10,2));
+  
+ALTER TABLE animals
```

Comment:
I would suggest you remove this as you already created a table for `species` below. 👌🏽